src/OpenTelemetry.Contrib.Instrumentation.GrpcCore/README.md

```diff
@@ -0,0 +1,47 @@
+# gRPC Core-based Client and Server Interceptors for OpenTelemetry .NET
+
+Adds OpenTelemetry instrumentation for gRPC Core-based client and service calls.
+
+gRPC Core is the predecessor to ASP.NET Core gRPC. See <https://github.com/grpc/grpc/tree/master/src/csharp>
+
+For ASP.NET Core gRPC instrumentation see <https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md>
```

Comment:
This is *only* for client side instrumentation. For server side, its "bundled" within the AspNetCore instrumentation. Would be good to call it out here.